lib/models/project.js

```diff
@@ -416,9 +446,54 @@ class Project {
     @method discoverAddons
    */
   discoverAddons() {
-    let addonsList = this.addonDiscovery.discoverProjectAddons(this);
+    console.log(" ");
+    console.log(`discoverAddons for root ${this.root}, pkg ${this.pkg.name}.`);
```

Comment:
before merging, these logs will have to be removed, or switched to use heimdall-logger/debug

Reply:
Yep, I've removed all console.XXX calls that I added (there are a couple in comments somebody else did earlier that I left alone).